docs/conf.py

```diff
@@ -309,6 +309,19 @@
 linkcheck_timeout = 180
 linkcheck_anchors = False
 
+# -- Options for Panel -------------------------------------------
+
+nbsite_pyodide_conf = {
```

Comment:
What does this do? A comment would be useful

Reply:
this was according to the panel documentation and to embed the widget on the tardis documentation. I added comments now!